apps/Dataman/src/screens/MaterialRequest.tsx

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+import { View, Text, List, Button } from "@ant-design/react-native";
+import { ScrollView, StyleSheet } from "react-native";
+import { AntDesign } from "@expo/vector-icons";
+import { Navbar } from "../components/Navbar";
+import { InputField } from "../components/InputField";
+import { PageHeader } from "../components/PageHeader";
+import { FormButton } from "../components/Button";
+
+const MaterialRequest = () => {
+  const Item = List.Item;
+
+  return (
+    <View style={{ height: "100%" }}>
+      <Navbar />
+      <PageHeader text="Material Request Entry" />
+
+      <ScrollView>
+        <List>
+          <Item arrow="horizontal">
+            <Item.Brief>
+              <Text style={styles.list_text}>Voucher Type</Text>
+              <AntDesign name="questioncircleo" />
+            </Item.Brief>
+            <Text style={styles.text}>Search Here</Text>
+          </Item>
+        </List>
+
+        <List>
```

Comment:
refactor this into an array containing list data and use .map() to iteratate over this list in order to create the list view

Reply:
Done